src/lib/Behat/PageObject/ChangePasswordPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageObject;
+
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\ElementFactory;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\RightMenu;
+
+class ChangePasswordPage extends Page
+{
+    /** @var string Name by which Page is recognised */
+    public const PAGE_NAME = 'ChangePassword';
+
+    /**
+     * @var RightMenu
+     */
+    public $rightMenu;
+
+    public function __construct(UtilityContext $context)
+    {
+        parent::__construct($context);
+        $this->rightMenu = ElementFactory::createElement($this->context, RightMenu::ELEMENT_NAME);
+        $this->siteAccess = 'admin';
+        $this->route = '/user/change-password';
+        $this->pageTitleLocator = '.ez-header h1';
+        $this->fields = [
+            'oldPassword' => '#user_password_change_oldPassword',
+            'newPassword' => '#user_password_change_newPassword_first',
+            'confirmPassword' => '#user_password_change_newPassword_second',
+        ];
+    }
+
+    public function verifyElements(): void
+    {
+        $this->rightMenu->verifyVisibility();
+    }
+
+    public function setOldPassword($value): void
+    {
+        $this->context->findElement($this->fields['oldPassword'])->setValue($value);
+    }
+
+    public function setNewPassword($value): void
```

Comment:
```suggestion
    public function setNewPassword(string $value): void
```

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1764/commits/3093c904caf093b598c734609c7ec09467bdf492.